.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
what is this ? !

Reply:
Visit https://EditorConfig.org It's a file that configures your IDE. It's supported by most of IDEs